pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -165,8 +165,13 @@ spec:
     */}}
     {{- $replicaCount := int64 .Config.ReplicaCount.value | saveAs "rc" .ListItems -}}
     {{- $poolsList := jsonpath .JsonResult "{range .items[?(@.status.phase=='Online')]}pkey=pools,{@.metadata.uid}={@.metadata.name};{end}" | trim | default "" | splitList ";" -}}
+    {{/*
+    The last item would always be an empty string so decrease the count by one
+    */}}
+    {{- $availablePools := len $poolsList -}}
```

Comment:
How about having our own splitlist that trims the delimiter from prefix and suffix before carrying out the spilt

Reply:
Sounds good. I will implement it.

Reply:
What if there is a string `';;;'` should this return `['']` or `['','']`.